### PR TITLE
website/nutaku.net

### LINKED ISSUE #2813: Nutaku | nutaku.net
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [x] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 579#3440 

**What's the name of the service?** www.nutaku.net 

**What should the Presence display?** The Presence should show what game people are playing.

**If possible, please provide a logo for the service (512x512 minimum)** 
logo nutaku

### PR DESCRIPTION
**Pull request by Venipa**
# nutaku.net (nsfw games website)

### Search Page
image
image

### Game Page
image

### Browser Game - Play Page
image

### News - All Articles
image

### News - Reading Article
image

### default presence
image

Resolves #2813

### CODE REVIEW

File: websites/N/Nutaku/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "service": "Nutaku",
+  "description": {
+    "en": "The best Hentai Games online on Nutaku. Your number 1 destination for uncensored browser, mobile and downloadable hentai games."
+  },
+  "regExp": "([a-z]+[.])nutaku.net[/]",
+  "url": ["nutaku.net", "www.nutaku.net"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/fvSnqjK.png",
+  "thumbnail": "https://i.imgur.com/h9zq9na.png",
```
Comment (ririxidev) [inline websites/N/Nutaku/dist/metadata.json:15]: could you change these images on screenshot to idk, our logo? 😂

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @Venipa!
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/N/Nutaku/dist/metadata.json (hunk 1 above)
Comment (Venipa) [reply websites/N/Nutaku/dist/metadata.json:15]: it is their logo, just picked their accent/primary color. its all in their branding

Comment (SlowLife1661) [reply websites/N/Nutaku/dist/metadata.json:15]: If I think correctly from what ririxi said, ririxi asking you to change the images on thumbnail to something like this
a

Comment (Venipa) [reply websites/N/Nutaku/dist/metadata.json]: [image](https://i.imgur.com/ypxOFV9.jpg)

Comment (ririxidev) [reply websites/N/Nutaku/dist/metadata.json]: good too 👍

**Review by ririxidev — APPROVED** [review]
lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**